### DISCUSSION TITLE
Proxy URL path https://myURL/mypath/ to localhost:8080/

### DISCUSSION THREAD

**Discussion by cyprus1spirit** [owncast/owncast#3442] [General Owncast discussion]
I only have a single URL on single URL domain that is serving my web site. My URL homepage is my site's homepage. It looks like Owncast does not like when I proxy a url path like:  "https://myURL/mystream/" to "localhost:8080/".

The page is shown but the stream and icons not. 

Any ideas?

**Comment by gabek** [comment]
I've never tried this. But I would guess you'd have to track down every path that isn't working and manually add it to your proxy, telling it where to proxy to. But honestly I'm not sure. This isn't how Owncast is built to be run, so there could be any number of issues.

**Reply by cyprus1spirit** [reply]
Thank you very much for your reply. I used subdomains and sorted the issue out.

**Comment by cyprus1spirit** [comment]
Owncast is a very good project built on top of Ant Media Server (the Community edition) to serve users who want something set up and run on the fly. I can advice towards keeping it as simple as possible, avoid unnecessary bells and whistles and keeping the initial philosophy through possible updates. 

If somebody wants multiple channels he can always use it as a service and run multiple service instances with it with different ports. 

It gets complicated to tweak  but in any case someone can use subdomains as a hard workaround.

Authentication of the stream is another discussion.  

Thanks very much for bringing this project to us.

**Reply by gabek** [reply]
> Owncast is a very good project built on top of Ant Media Server (the Community edition)

Owncast has nothing to do with Ant Media Server. It's not built on top of anything.
